src/components/Header.tsx

```diff
@@ -30,96 +43,44 @@ interface HeaderProps {
-    setDate(setYear(date, parseInt(event.target.value)));
-  };
+  const getMonth = () => MONTHS[parseOptionalDate(date, date).getMonth()];
+  const getYear = () => parseOptionalDate(date, date).getFullYear();
 
   return (
     <Grid container justify="space-between" alignItems="center">
-      <Grid item className={classes.iconContainer}>
-        <IconButton className={classes.icon} disabled={prevDisabled} onClick={onClickPrevious}>
-          <ChevronLeft color={prevDisabled ? "disabled" : "action"} />
-        </IconButton>
-      </Grid>
-      <Grid item>
-        <Select
-          value={getMonth(date)}
-          onChange={handleMonthChange}
-          MenuProps={{
-            anchorOrigin: {
-              vertical: "bottom",
-              horizontal: "left",
-            },
-            transformOrigin: {
-              vertical: "top",
-              horizontal: "left",
-            },
-            getContentAnchorEl: null,
-          }}
-          className={classes.header}
-        >
-          {MONTHS.map((month, idx) => (
-            <MenuItem key={month} value={idx} className={classes.header}>
-              {month}
-            </MenuItem>
-          ))}
-        </Select>
-      </Grid>
+      {prevDisabled || (
+        <Grid item className={classes.iconContainer}>
+          <IconButton className={classes.icon} disabled={prevDisabled} onClick={onClickPrevious}>
+            <ChevronLeft color={prevDisabled ? "disabled" : "action"} />
+          </IconButton>
+        </Grid>
+      )}
 
-      <Grid item>
-        <Select
-          value={getYear(date)}
-          onChange={handleYearChange}
-          MenuProps={{
-            anchorOrigin: {
-              vertical: "bottom",
-              horizontal: "left",
-            },
-            transformOrigin: {
-              vertical: "top",
-              horizontal: "left",
-            },
-            getContentAnchorEl: null,
-          }}
-          className={classes.header}
-        >
-          {generateYears(date, 30).map((year) => (
-            <MenuItem key={year} value={year} className={classes.header}>
-              {year}
-            </MenuItem>
-          ))}
-        </Select>
-
-        {/* <Typography>{format(date, "MMMM YYYY")}</Typography> */}
-      </Grid>
-      <Grid item className={classes.iconContainer}>
-        <IconButton className={classes.icon} disabled={nextDisabled} onClick={onClickNext}>
-          <ChevronRight color={nextDisabled ? "disabled" : "action"} />
-        </IconButton>
+      <Grid item className={combine(classes.flexSpace, prevDisabled ? classes.paddingLeft : "")}>
+        <span className={classes.monthHeader}>{getMonth()}</span>
+        <span> </span>
+        <span className={classes.monthHeader}>{getYear()}</span>
       </Grid>
+
+      {nextDisabled || (
+        <Grid item className={classes.iconContainer}>
+          <IconButton className={classes.icon} disabled={nextDisabled} onClick={onClickNext}>
```

Comment:
If `nextDisabled` is true, the component will not be rendered, hence there is not need to disable the button.